src/Microsoft.EntityFramework.SqlServer/readme.md

```diff
@@ -0,0 +1,138 @@
+
+## Configuration
+
+There are various ways to configure Entity Framework to use this provider.
+
+You can register the provider in code using an attribute:
+
+````csharp
+[DbConfigurationType(typeof(MicrosoftSqlDbConfiguration))]
+public class SchoolContext : DbContext
+{
+    public SchoolContext() : base()
+    {
+    }
+
+    public DbSet<Student> Students { get; set; }
+}
+````
+If you have multiple classes inheriting from DbContext in your solution, add the DbConfigurationType attribute to all of them.
+
+Or you can use the SetConfiguration method before any data access calls:
+````csharp
+ DbConfiguration.SetConfiguration(new MicrosoftSqlDbConfiguration());
+````
+Or you can add the following lines to your existing DbConfiguration class:
+````csharp
+SetProviderFactory(MicrosoftSqlProviderServices.ProviderInvariantName, Microsoft.Data.SqlClient.SqlClientFactory.Instance);
+SetProviderServices(MicrosoftSqlProviderServices.ProviderInvariantName, MicrosoftSqlProviderServices.Instance);
+// Optional
+SetExecutionStrategy(MicrosoftSqlProviderServices.ProviderInvariantName, () => new MicrosoftSqlAzureExecutionStrategy());
+````
+You can also use App.Config based configuration:
+
+````xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <configSections>
+        <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />    
+    </configSections>
+    <entityFramework>
+        <providers>		
+            <provider invariantName="Microsoft.Data.SqlClient" type="System.Data.Entity.SqlServer.MicrosoftSqlProviderServices, Microsoft.EntityFramework.SqlServer" />
+        </providers>
+    </entityFramework>
+    <system.data>
+        <DbProviderFactories>
+           <add name="SqlClient Data Provider"
+             invariant="Microsoft.Data.SqlClient"
+             description=".NET Framework Data Provider for SqlServer"
+             type="Microsoft.Data.SqlClient.SqlClientFactory, Microsoft.Data.SqlClient" />
+        </DbProviderFactories>
+    </system.data>
+</configuration>
+````
+If you use App.Config with a .NET 6 or later app, you must register the DbProviderFactory in code once:
+
+````csharp
+DbProviderFactories.RegisterFactory(MicrosoftSqlProviderServices.ProviderInvariantName, Microsoft.Data.SqlClient.SqlClientFactory.Instance);
+````
+
+## EDMX usage
+
+If you use an EDMX file, update the `Provider` name:
+
+````xml
+<edmx:Edmx Version="3.0" xmlns:edmx="http://schemas.microsoft.com/ado/2009/11/edmx">
+  <edmx:Runtime>
+    <edmx:StorageModels>
+      <Schema Namespace="ChinookModel.Store" Provider="Microsoft.Data.SqlClient" >
+````
+
+> In order to use the EDMX file with the Visual Studio designer, you must switch the provider name back to `System.Data.SqlClient`
```

Comment:
I wonder if there's some clever solution we can document that works with our current [.NET Core workaround](https://learn.microsoft.com/en-us/ef/ef6/what-is-new/#ef-designer-support) and uses something like [XmlPoke](https://learn.microsoft.com/en-us/visualstudio/msbuild/xmlpoke-task?view=vs-2022).